app/forms/questionnaire_step1_form.rb

```diff
@@ -0,0 +1,173 @@
+class QuestionnaireStep1Form
+  include ActiveModel::Model
+
+  attr_accessor :firm_email_address,
+                :firm_telephone_number,
+                :main_office_line_1,
+                :main_office_line_2,
+                :main_office_town,
+                :main_office_county,
+                :main_office_postcode,
+                :accept_customers_from,
+                :advice_in_person,
+                :advice_by_other_methods,
+                :free_initial_meeting,
+                :initial_meeting_duration,
+                :initial_advice_fee_structure,
+                :ongoing_advice_fee_structure,
+                :allow_customers_to_pay_for_advice,
+                :minimum_fixed_one_off_fee_amount
+
+  validates :firm_email_address,
+            presence: true,
+            length: {maximum: 50},
+            format: {with: /.+@.+\..+/}
+
+  validates :firm_telephone_number,
+            presence: true,
+            length: {maximum: 30},
+            format: {with: /\A[0-9 ]+\z/}
+
+  validates :main_office_line_1,
+            presence: true,
+            length: {maximum: 100}
+
+  validates :main_office_postcode,
+            presence: true,
+            format: {with: /\A[a-zA-Z\d]{1,4} [a-zA-Z\d]{1,3}\z/}
+
+  validates :main_office_line_2,
+            :main_office_town,
+            :main_office_county,
+            presence: true
+
+  validate :accept_customers_from_list_is_valid
+  validate :advice_in_person_list_is_valid
+  validate :advice_by_other_methods_list_is_valid
+  validate :free_initial_meeting_choice_is_valid
+  validate :initial_meeting_duration_choice_is_valid
+  validate :initial_advice_fee_structure_choice_is_valid
+  validate :ongoing_advice_fee_structure_choice_is_valid
+  validate :allow_customers_to_pay_for_advice_choice_is_valid
+
+  def accept_customers_from_options
```

Comment:
As we discussed, the lookup data here and elsewhere should stay in a form object, once we move most of this logic into the AR model.